Code/Tools/AssetProcessor/native/ui/SourceAssetDetailsPanel.cpp

```diff
@@ -50,75 +52,152 @@ namespace AssetProcessor
         }
         AssetTreeItem* childItem = static_cast<AssetTreeItem*>(sourceModelIndex.internalPointer());
 
-        const AZStd::shared_ptr<const SourceAssetTreeItemData> sourceItemData = AZStd::rtti_pointer_cast<const SourceAssetTreeItemData>(childItem->GetData());
+        const AZStd::shared_ptr<const SourceAssetTreeItemData> sourceItemData =
+            AZStd::rtti_pointer_cast<const SourceAssetTreeItemData>(childItem->GetData());
 
         m_ui->assetNameLabel->setText(childItem->GetData()->m_name);
 
         if (childItem->GetData()->m_isFolder || !sourceItemData)
         {
             // Folders don't have details.
-            SetDetailsVisible(false);
+            SetDetailsVisible(false, false);
```

Comment:
nit: When you have a string of boolean flags, consider making constexpr or similar so you can add semantic identifiers to the flags to aid future maintainers.

Reply:
Removed the second flag, it wasn't needed based on your other feedback